x-pack/plugins/monitoring/common/constants.ts

```diff
@@ -123,12 +123,12 @@ export const CLUSTER_ALERTS_ADDRESS_CONFIG_KEY = 'cluster_alerts.email_notificat
 
 export const STANDALONE_CLUSTER_CLUSTER_UUID = '__standalone_cluster__';
 
-export const INDEX_PATTERN = '.monitoring-*-6-*,.monitoring-*-7-*';
-export const INDEX_PATTERN_KIBANA = '.monitoring-kibana-6-*,.monitoring-kibana-7-*';
-export const INDEX_PATTERN_LOGSTASH = '.monitoring-logstash-6-*,.monitoring-logstash-7-*';
-export const INDEX_PATTERN_BEATS = '.monitoring-beats-6-*,.monitoring-beats-7-*';
-export const INDEX_ALERTS = '.monitoring-alerts-6*,.monitoring-alerts-7*';
-export const INDEX_PATTERN_ELASTICSEARCH = '.monitoring-es-6-*,.monitoring-es-7-*';
+export const INDEX_PATTERN = '.monitoring-*';
```

Comment:
Note: isn't this pattern also encompassing the other four?

Reply:
I wonder what was the reasoning behind the hardcoded versions here ? Could we have some legacy .monitoring version 5 out there that we don't support anymore ?

Reply:
fair thing to wonder. Maybe @jasonrhodes or @chrisronline have some idea?

Reply:
I spun up a 5.6.16 cluster to test and the monitoring indices are `-6` interestingly enough.

![Screen Shot 2021-12-22 at 15 26 32](https://user-images.githubusercontent.com/690/147045962-3a1ef9a9-71e4-4bca-95a8-5988f10daf71.png)

I guess I could just add another comma and add 8 if we want to play it safe. But if the 5.x stack is also `6` and the last 5 release was EOL `2019-03-11`, it might be reasonable to remove the version now.